allennlp/common/registrable.py

```diff
@@ -48,9 +48,9 @@ def register(cls: Type[T], name: str, exist_ok=False):
 
         Parameters
         ----------
-        name : ``str``
+        name: ``str``
             The name to register the class under.
-        exist_ok : ``bool`, optional (default=False)
+        exist_ok: ``bool`, optional (default=False)
```

Comment:
You just need to add one more backtick here - it's lopsided, that's why sphinx is complaining.

Reply:
Yep--sorry I missed this comment so I didn't fix this (but it isn't a regression from this PR).